charts/kubebb-core/templates/deployment.yaml

```diff
@@ -40,10 +46,19 @@ spec:
           value: /opt/helm/config/helm
         - name: HELM_DATA_HOME
           value: /opt/helm/data/helm
+{{- if .Values.webhook.enable }}
+        - name: ENABLE_WEBHOOKS
+          value: true
```

Comment:
should use `"true"` instead of boolean here  due to error: 

```shell
Error: UPGRADE FAILED: failed to create resource: Deployment in version "v1" cannot be handled as a Deployment: json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string
```